<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraPreview</class>
 <widget class="QDialog" name="CameraPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LeonCam</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Resources/Images/logo.ico</normaloff>:/Resources/Images/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#CameraPreview
{
	background-image: url(:/Resources/Images/backgroundMain.png);
}</string>
  </property>
  <widget class="QLabel" name="LCameraDetails">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;
font-weight:600;</string>
   </property>
   <property name="text">
    <string>Camera details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBPatrol">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>390</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Patrol</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBRecognize">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Recognize</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>390</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBSnapshot">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>390</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#PBSnapshot{
background-image: url(:/Resources/Images/snapshot.png);
border: none;
margin: 0px;
padding: 0px;
}

#PBSnapshot:hover{
background-image: url(:/Resources/Images/snapshotHover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LPreviewScreen">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>640</width>
     <height>360</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LDetectedPeople">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;
font-weight:600;</string>
   </property>
   <property name="text">
    <string>Number of detected people: 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBHome">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>310</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#PBHome{
background-image: url(:/Resources/Images/home.png);
border: none;
margin: 0px;
padding: 0px;
}

#PBHome:hover{
background-image: url(:/Resources/Images/homeHover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBUp">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>287</y>
     <width>30</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#PBUp {
background-image: url(:/Resources/Images/up.png);
border: none;
margin: 0px;
padding: 0px;}
#PBUp:hover{
background-image: url(:/Resources/Images/upHover.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBRight">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>315</y>
     <width>23</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#PBRight {
background-image: url(:/Resources/Images/right.png);
border: none;
margin: 0px;
padding: 0px;}
#PBRight:hover{
background-image: url(:/Resources/Images/rightHover.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBDown">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>350</y>
     <width>30</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#PBDown {
background-image: url(:/Resources/Images/down.png);
border: none;
margin: 0px;
padding: 0px;}
#PBDown:hover{
background-image: url(:/Resources/Images/downHover.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBLeft">
   <property name="geometry">
    <rect>
     <x>627</x>
     <y>315</y>
     <width>23</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#PBLeft {
background-image: url(:/Resources/Images/left.png);
border: none;
margin: 0px;
padding: 0px;}
#PBLeft:hover{
background-image: url(:/Resources/Images/leftHover.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBSavePreset">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CBPresets">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PBBack">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#PBBack{
color:rgb(255, 255, 255);
background-color: rgb(255, 77, 61);}
#PBBack:hover{
background-color: rgb(255, 87, 58);
}
/*border: none;
margin: 0px;
padding: 0px;
background-color:none*/</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBCameraOnOff">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>On/Off</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBGoToPreset">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Go To</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
